src/Compilers/CSharp/Test/Symbol/Symbols/Source/RecordTests.cs

```diff
@@ -15,18 +14,22 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
     public class RecordTests : CompilingTestBase
     {
         private static CSharpCompilation CreateCompilation(CSharpTestSource source)
-            => CSharpTestBase.CreateCompilation(source, parseOptions: TestOptions.RegularPreview);
+            => CSharpTestBase.CreateCompilation(new[] { source, IsExternalInitTypeDefinition }, parseOptions: TestOptions.RegularPreview);
 
         private CompilationVerifier CompileAndVerify(CSharpTestSource src, string? expectedOutput = null)
-            => base.CompileAndVerify(src, expectedOutput: expectedOutput, parseOptions: TestOptions.RegularPreview);
+            => base.CompileAndVerify(new[] { src, IsExternalInitTypeDefinition },
+                expectedOutput: expectedOutput,
+                parseOptions: TestOptions.RegularPreview,
+                // init-only fails verification
```

Comment:
>// init-only fails verification [](start = 16, length = 31)

I think it only fails verification on desktop. I thought it's passing on core.

`verify: ExecutionConditionUtil.IsCoreClr ? Verification.Passes : Verification.Fails);`

Reply:
I'm pretty sure we only have a desktop PEVerify. It's probably passing because we don't do anything.